### PR TITLE
Added EmojiPicker

### PR DESCRIPTION
**Pull request by jsonjuri**
- Added EmojiPicker
- Combined Stickers with Emojis

### CODE REVIEW

**Comment by t3chguy** [comment]
Any screenshots of how this looks? Nad has very specific plans on how he wants this to look which is why https://github.com/matrix-org/matrix-react-sdk/pull/3747 is blocked on design from him, it is likely this will get further stuck.

**Comment by jsonjuri** [comment]
My design looks a bit different but it will look fine in the default theme, not so many CSS has been added.

EmojiPanel
image


Stickers Panel
image

**Comment by t3chguy** [comment]
Seems like you leaked your other changes into this PR.

**Comment by jsonjuri** [comment]
Somehow it got mixed up let me know if I need to make a new pull request?

File: src/components/views/rooms/StickerEmojiPicker.js [hunk 1]
```diff
@@ -0,0 +1,511 @@
+/*
+Copyright 2018 New Vector Ltd
```
Comment (t3chguy) [inline src/components/views/rooms/StickerEmojiPicker.js]: This isn't right.

**Pull request closed without merging**